include/gsl/gsl

```diff
@@ -109,6 +109,13 @@ private:
     T ptr_;
 };
 
+template <class T>
```

Comment:
You should add `#include <iostream>` in order to use `std::ostream` reliably here.